### PR TITLE
Get conda package version from setup.py.

### PR DESCRIPTION
**Pull request by ceball**
Demo

(will fail because conda version can't have hyphen in it)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ Autover provides:
      location for each Python package specified; see the [autover
      script](https://github.com/ioam/autover/blob/master/scripts/autover).
 
-Authors: Jean-Luc Stevens, Chris Ball and James A. Bednar
+Authors: Jean-Luc Stevens and James A. Bednar
```
Comment (jlstevens) [inline README.md:13]: Sneaky! ;-p

Comment (ceball) [reply README.md:13]: I'm sure Jim didn't appreciate having his name so close to a missing comma.

Comment (jlstevens) [reply README.md:13]: Ah, that is a very good point!

**Comment by ceball** [comment]
Replaced by #17

**Pull request closed without merging**